ingestion/src/metadata/orm_profiler/api/workflow.py

```diff
@@ -65,20 +63,13 @@ def __init__(self, config: OpenMetadataWorkflowConfig):
             self.config.workflowConfig.openMetadataServerConfig
         )
 
-        # We will use the existing sources to build the Engine
-        self.source: Source = get_ingestion_source(
-            source_type=self.config.source.type,
-            source_config=self.config.source,
-            metadata_config=self.metadata_config,
-        )
-
-        if not isinstance(self.source, SQLSource):
-            raise ValueError(
-                f"Invalid source type for {self.source}. We only support SQLSource in the Profiler"
-            )
+        # Prepare the connection to the source service
+        # We don't need the whole Source class, as it is the OM Server
+        engine = get_engine(self.config.source.serviceConnection.__root__.config)
+        test_connection(engine)
```

Comment:
Profiler workflow now directly loads the engine, instead of relying on building a "fake" source. The Source will always be OM REST for the profiler, so this helps just get the piece if info required